src/particles/elements/Quad.H

```diff
@@ -58,17 +58,29 @@ namespace impactx
             amrex::ParticleReal const pt_ref = refpart.pt;
             amrex::ParticleReal const betgam2 = pow(pt_ref, 2) - 1.0_prt;
 
-            // advance position and momentum (drift)
-            p.pos(0) = cos(m_k*m_ds)*x + sin(m_k*m_ds)/m_k*px;
-            // px = px;
-            px = -m_k*sin(m_k*m_ds)*x + cos(m_k*m_ds)*px;
+            amrex::ParticleReal const omega = sqrt(abs(m_k));
```

Comment:
can you add a comment about how we call this term usually?

Reply:
Ok, I will add a comment here.

Reply:
Done.  See above.

Reply:
Awesome, thanks a lot! :)